src/auth/jwt.rs

```diff
@@ -0,0 +1,98 @@
+use {
+    relay_rpc::{
+        auth::{
+            JwtClaims,
+            JwtHeader,
+            JwtVerificationError,
+            DID_DELIMITER,
+            DID_METHOD,
+            DID_PREFIX,
+            JWT_DELIMITER,
+        },
+        domain::{ClientId, DecodedClientId},
+    },
+    std::collections::HashSet,
+};
+
+#[derive(Debug)]
+pub struct Jwt(pub String);
+
+impl Jwt {
+    pub fn decode(&self, aud: &HashSet<String>) -> Result<ClientId, JwtVerificationError> {
```

Comment:
I think this was put into the walletconnect rust lib

Reply:
Is it? Will take a look

Reply:
Thanks :)